.github/workflows/format-check.yml

```diff
@@ -0,0 +1,41 @@
+name: format-check
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags: '*'
+  pull_request:
```

Comment:
IMO the only thing that is needed is a check that is run in every PR. Maybe just copy common setups such as in https://github.com/JuliaDiff/ChainRulesCore.jl/blob/main/.github/workflows/format.yml?

Reply:
Oh nice. I didn't know about `reviewdog/action-suggester`. Just refactored to be like the `ChainRulesCore` workflow.